src/traces/scatter3d/convert.js

```diff
@@ -331,11 +331,11 @@ function convertPlotlyOptions(scene, data) {
     return params;
 }
 
-function arrayToColor(color) {
-    if(Array.isArray(color)) {
+function _arrayToColor(color) {
+    if(Lib.isArrayOrTypedArray(color)) {
         var c = color[0];
 
-        if(Array.isArray(c)) color = c;
+        if(Lib.isArrayOrTypedArray(c)) color = c;
```

Comment:
are you sure you still need this line?

Reply:
Yes. That is to support typed arrays in `scatter3d`.